components/FlashCard.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { useState } from "react";
+import FlashCardFront from "./FlashCardFront";
+import FlashCardBack from "./FlashCardBack";
+import Button from "./Button";
+
+const StyledCard = styled.li`
+  background-color: ${({ $showAnswer }) =>
+    $showAnswer ? "#93E9BE" : "lightgray"};
+  list-style: none;
+  width: 80%;
+  border-radius: 20px;
+  padding: 25px 25px;
+  line-height: 1.25;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+const CardInner = styled.div``;
+
+export default function FlashCard({
+  question,
+  answer,
+  collectionId,
+  onMarkCorrect,
+  id,
+  flashCards,
+}) {
+  const [showAnswer, setShowAnswer] = useState(false);
+
+  function flipCard() {
+    setShowAnswer(!showAnswer);
+  }
+
+  function handleCorrect(event) {
+    event.stopPropagation();
+    onMarkCorrect(id);
+  }
+
+  return (
+    <StyledCard $showAnswer={showAnswer} onClick={flipCard}>
+      <CardInner>
+        {showAnswer ? (
+          <>
+            <Button
+              name={
+                flashCards.find((flashCard) => flashCard.id === id)?.isCorrect
```

Comment:
instead of passing all cards into this component you should pass down only the single one so you don't have to search through the list here.
```suggestion
                flashCard.isCorrect
```